packages/server/tests/views/home-feed.test.ts

```diff
@@ -62,9 +61,6 @@ describe('pds home feed views', () => {
 
     expect(forSnapshot(aliceFeed.data.feed)).toMatchSnapshot()
     aliceFeed.data.feed.forEach(expectOriginatorFollowedBy(alice))
-    expect(getCursors(aliceFeed.data.feed)).toEqual(
```

Comment:
is there a reason these tests are taken out here & not on views like likes/follow/reposts?

Reply:
The reason I removed these is because the value used for pagination doesn't exist on individual feed items anymore.  The users search view is another one like this.  While it would be awesome to test the property explicitly like this, my thought/hope is that the snapshots technically still cover this for us.